src/poprox_storage/repositories/accounts.py

```diff
@@ -47,6 +48,7 @@ def fetch_accounts(self, account_ids: list[UUID] | None = None) -> list[Account]
                 email=row.email,
                 status=row.status,
                 source=row.source,
+                subsource=row.subsource,
```

Comment:
I think this change might conflict with a recent change that I made, it's for the better, but it'll probably be one of those merges where this change has to be thrown out and "re-applied" against a different baseline. It should be easy to do and I'm happy to help.

Reply:
I don't think we have an answer to this question as asked. 

Did you see the change that got merged earlier (the one that's marked as a merge conflict with this PR) I didn't _massively_ change the field list, the big thing I did is changed some architecture to move towards the same code patterns seen in other repos. This change makes a single function in charge of building most `Account` objects in practice. That change is for the better, but it'll conflict here since it moves the code around quite a bit.

Realistically, I _think_ we can then just do the code version of a select * from the table and let the code that builds the `Account` objects grab every row it wants/needs. That way we get a consistent property set.